test/algorithms/set_operations/difference/difference_linear_linear.cpp

```diff
@@ -548,61 +511,45 @@ BOOST_AUTO_TEST_CASE( test_difference_linestring_linestring )
          "lldf31s"
          );
 
-    tester::apply
-        (from_wkt<L>("LINESTRING(8 1, 4 .4,2 8)"),
-         from_wkt<L>("LINESTRING(0 -.2, 8 1)"),
-         from_wkt<ML>("MULTILINESTRING((4 .4,2 8))"),
-         "lldf31x",
-         1e-10
-         );
-
-    tester::apply
-        (from_wkt<L>("LINESTRING(2 8,4 .4,8 1)"),
-         from_wkt<L>("LINESTRING(0 -.2, 8 1)"),
-         from_wkt<ML>("MULTILINESTRING((2 8,4 .4))"),
-         "lldf31x-r",
-         1e-10
-         );
-
-    tester::apply
-        (from_wkt<L>("LINESTRING(0 5, 8 1, 4 .4, 2 8)"),
-         from_wkt<L>("LINESTRING(0 -.2, 8 1, -.5 7)"),
-         from_wkt<ML>("MULTILINESTRING((0 5,8 1),(4 .4,2 8))"),
-         "lldf31y",
-         1e-10
-         );
-
-    tester::apply
-        (from_wkt<L>("LINESTRING(0 -.2, 8 1, -.5 7)"),
-         from_wkt<L>("LINESTRING(0 5, 8 1, 4 .4, 2 8)"),
-         from_wkt<ML>("MULTILINESTRING((0 -.2,4 .4),(8 1,-.5 7))"),
-         "lldf31y-r",
-         1e-10
-         );
-
-    tester::apply
-        (from_wkt<L>("LINESTRING(0 5, 8 1, 4 .4, 2 8)"),
-         from_wkt<L>("LINESTRING(0 -.2, 8 1, -.5 7, 6 +.2)"),
-         from_wkt<ML>("MULTILINESTRING((0 5,8 1),(4 .4,2 8))"),
-         "lldf31y+",
-         1e-10
-         );
-
-    tester::apply
-        (from_wkt<L>("LINESTRING(10.0002 2,9 -1032.34324, .3 8, 0 5, 8 1, 4 .4, 2 8)"),
-         from_wkt<L>("LINESTRING(0 -.2, 8 1, -.5 7, 6 +.2)"),
-         from_wkt<ML>("MULTILINESTRING((10.0002 2,9 -1032.34324,.3 8,0 5,8 1),(4 .4,2 8))"),
-         "lldf31z",
-         1e-10
-         );
+    {
+        ut_settings settings{1e-10};
+#if ! defined(BOOST_GEOMETRY_USE_RESCALING) && ! defined(BOOST_GEOMETRY_TEST_FAILURES)
+        // Cases failing without rescaling when first linestring is reversed
+        settings.test_reverse_normal = false;
+        settings.test_reverse_reverse = false;
```

Comment:
@vissarion for example you can see here that only two of the cases are excluded from the test, and only for if rescaling is not used. So in the default case, everything is still tested as it was before (now without duplicates)